config/testgrids/config.yaml

```diff
@@ -2585,559 +2071,6 @@ dashboards:
 - name: presubmits-kube-batch
 - name: presubmits-node-problem-detector
 - name: google-rules_k8s
-- name: istio-presubmits
```

Comment:
This dashboard is deleted, but not present in your configuration.

Was this intentional? Is `istio-presubmits` spelled correctly in the generating yaml doc?

Reply:
This is intentional, these are legacy from before we split tests by branch